docker-compose.yml

```
@@ -36,16 +36,16 @@ services:
       # Configured gateway routing fees Format: <base_msat>,<proportional_millionths>
       - FM_GATEWAY_FEES=0,1000
       # LND RPC address
-      - FM_LND_RPC_ADDR=10.5.0.10:11009
+      - FM_LND_RPC_ADDR=https://lnd_1:8180
```

We use a domain name for this since we add the domain in lnd.conf for TLS generation. Might not be necessary, but this got me to the next error: 
```
ui-gatewayd_1-1   | 2023-07-06T02:37:06.925764Z  WARN ln_gateway::lnd: Couldn't connect to LND, retrying in 1 second... LndConnectError { internal: Connect { address: "https://lnd_
1:8180", error: tonic::transport::Error(Transport, hyper::Error(Connect, H2NotNegotiated)) } }
```
Seems to be http2 related.

---

Switching back to 11009 fixes this for me, gatewayd is now working
```suggestion
      - FM_LND_RPC_ADDR=https://lnd_1:11009
```